<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calendar::Internal::ItemEditorWidget</class>
 <widget class="QWidget" name="Calendar::Internal::ItemEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>473</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="11" column="0">
    <layout class="QVBoxLayout" name="endingLayout"/>
   </item>
   <item row="5" column="0">
    <layout class="QVBoxLayout" name="afterDateTimeLayout">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="beforeDateTimeLayout">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Date, time and label</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="spacing">
       <number>-1</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Duration</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QComboBox" name="durationCombo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>End</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4" colspan="2">
       <widget class="QCheckBox" name="busyCheck">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Show busy</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QDateEdit" name="startDate">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QDateEdit" name="endDate">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QTimeEdit" name="startTime"/>
      </item>
      <item row="3" column="3">
       <widget class="QTimeEdit" name="endTime"/>
      </item>
      <item row="5" column="0" colspan="6">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="password">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1" colspan="5">
       <widget class="QLineEdit" name="eventLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Label</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QComboBox" name="statusCombo"/>
      </item>
      <item row="4" column="4" colspan="2">
       <widget class="QCheckBox" name="privateCheck">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Is a private</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Extra</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="beforeGeneralInformationLayout">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="generalInformationGroup">
         <property name="title">
          <string>General information</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Type</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="typeCombo"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Location</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="location"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Calendar</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QComboBox" name="calendarCombo"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QVBoxLayout" name="afterGeneralInformationLayout">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Description</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="beforeDescriptionLayout"/>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="descriptionGroup">
         <property name="title">
          <string>Extended description</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Full information</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QTextEdit" name="fullInfo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Icon</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="iconLabel">
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="selectIconButton">
            <property name="text">
             <string>Select icon</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QVBoxLayout" name="afterDescriptionLayout"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="beginningLayout">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>eventLabel</tabstop>
  <tabstop>startDate</tabstop>
  <tabstop>startTime</tabstop>
  <tabstop>durationCombo</tabstop>
  <tabstop>endDate</tabstop>
  <tabstop>endTime</tabstop>
  <tabstop>busyCheck</tabstop>
  <tabstop>statusCombo</tabstop>
  <tabstop>privateCheck</tabstop>
  <tabstop>password</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>typeCombo</tabstop>
  <tabstop>location</tabstop>
  <tabstop>calendarCombo</tabstop>
  <tabstop>fullInfo</tabstop>
  <tabstop>selectIconButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
